src/SystemHandle.cpp

```diff
@@ -301,4 +297,6 @@ class SystemHandle : public virtual FullSystem
 } //  namespace is
 } //  namespace eprosima
 
+// TODO aliases must come from CMAKE
 IS_REGISTER_SYSTEM("fastdds", eprosima::is::sh::fastdds::SystemHandle)
+IS_REGISTER_SYSTEM("databroker", eprosima::is::sh::fastdds::SystemHandle)
```

Comment:
This is no longer necessary if the Integration Service core PR is not going to be merged

Reply:
:+1: